lib/kahuna_client/client/server.rb

```diff
@@ -12,15 +12,26 @@ module Server
       # event (e.g. event = start) Event Name
       # user_info (e.g. user_info = {'first_name': 'John', 'last_name': 'Doe', 'gender': 'm'})
       def send_event(options = {})
+        events = if options[:events]
+                       options[:events].each do |key, obj|
```

Comment:
a little confused on what the input is vs what the output is. can you give example of what this is trying to accomplish?

also i think L18 references a variable that is not defined